adal/changelog.txt

```diff
@@ -1,3 +1,16 @@
+Version 4.4.0
+===============
+Cache fixes 
+   - Not remove account cache entities [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1365)
+   - Fix issue with sharing in memory cahce for different instances of the TokenCache [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1360)
+   - Clear in memory version of cache [MSAL issue #654](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/654)
+   - Fix NullPointException when adalv3 account is used because there is no client info in adalv3 [MSAL Issue #653](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/654)
+   - RemoveAdalUser api now removes users by accountID and by displayableID to cover migration scenario from adalv3 to adalv4 to msalv2. [MSAL Issue #652](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/652)
+   - Fix for internal TokenCache.Clear api so ADAL persistent cache is also cleared [MSAL Issue #651](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/651)
+   - Optimize cache lookups with native iOS by modifying field for persistence on iOS. [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1350)
+   - Fix for reading ExtraQueryParameter environment variable on netcore in ADAL [ADAL issue #1362](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1362)
+- Make a ConfidentialClient entry point obsolete on platforms which only support confidential clients [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1375)
```

Comment:
>-  [](start = 0, length = 2)

Cache fixes and this one should have the same indentation method, imo

Reply:
sounds good.

---
In reply to: [231366029](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1389#discussion_r231366029) [](ancestors = 231366029)

Reply:
I would propose:
Version 4.4.0
===============
Improvements and fixes to the token cache
- The serialized token cache can now be shared by different applications, therefore providing SSO if the same user signs-in in both applications
  - See [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1365) and [MSAL Issue #653](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/654)
- On .NET Core, the Token cache was shared by all instances of applications in memory. This is now fixed (See MSAL.NET issue #656 and [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1360))
- Fixes consistency issues for advanced token cache migration scenarios from ADAL v3.x to ADA v4.x to MSAL v2.x 
  - [MSAL Issue #652](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/652)
  - [MSAL Issue #651](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/651)
- Cache lookups were optimized. Work done in conjunction with ADAL.iOS and MSAL.iOS native) [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1350)
- The ExtraQueryParameter environment variable is now read on ADAL for the .NET Core platform, as was already the case in the .NET Framework case [ADAL issue #1362](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1362)
- A Client credential API which should not be used on Xamarin.iOS, Xamarin.Android and UWP is now marked as obsolete 
  - [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1375)

Reply:
perfect...thank you.

---
In reply to: [231409302](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1389#discussion_r231409302) [](ancestors = 231409302)